protobuf/v1/nexus.proto

```diff
@@ -108,6 +108,7 @@ message Child {
   ChildState state = 2;               // state of the child
   ChildStateReason state_reason = 3;  // child state reason
   int32 rebuild_progress = 4;         // rebuild progress
+  optional string device_name = 5;    // child device name
```

Comment:
what is the device_name? Should we add this to v0 as well?

Reply:
It's the name of the underlying block device (bdev name). There was no easy way to get that. Probably yes, makes sense to add it to v0.
I need it for my component level tests.

Reply:
I added to v0.